File: java/Constructor/Default_constructor/Ex2.java
class Mobile {
    String brand;
    int price;

    Mobile() {   // default constructor
        brand = "Samsung";
        price = 70000;
    }
}

public class Ex2 {
    public static void main(String[] args) {
        Mobile m = new Mobile();
        System.out.println(m.brand);
        System.out.println(m.price);
    }
}

File: c++/Constructor/Default_constructor/Ex2.cpp
#include <iostream>
using namespace std;

class Mobile {
public:
    string brand;
    int price;

    Mobile() {   // default constructor
        brand = "Samsung";
        price = 70000;
    }
};

int main() {
    Mobile m;
    cout << m.brand << endl;
    cout << m.price << endl;
}
